app.js

```diff
@@ -55,55 +57,36 @@ app.post('/api/ship-po', (req, res) => {
     
 });
 
+app.post('/api/sawtooth/create-po', (req, res) => {
+    console.log(req.body.items);
+    submitCreatePO(req.body.poNumber, req.body.items)
+    .then(function( response) {
+        res.status(200).send(response);
+    }, function (error) {
+        res.status(500).send(error);
+    })
+});
+
+app.post('/api/sawtooth/ship-po', (req, res) => {
+    console.log(req.body.poNumber);
+    submitShipPO(req.body.poNumber)
+    .then(function( response) {
+        res.status(200).send(response);
+    }, function (error) {
+        res.status(500).send(error);
+    })
+    
+});
+
 const PORT = 5001;
```

Comment:
doesn't it make sense to make the port configurable, rather than hard-coding it?